lib/rspec/core/bisect/runner.rb

```diff
@@ -89,6 +89,10 @@ def run_command(cmd)
           # :nocov:
         end
 
+        def spec_opts_without_bisect
+          { 'SPEC_OPTS' => ENV.fetch('SPEC_OPTS', '').gsub(/--bisect/, '') }
```

Comment:
What if the user has put `--bisect=verbose` in `SPEC_OPTS`?  This gsub will leave `=verbose`.